modules/client/jvm/src/test/scala/com/azavea/stac4s/api/client/StacClientSpec.scala

```diff
@@ -0,0 +1,279 @@
+                          |            "YY":6,
+                          |            "Run":3,
+                          |            "Tape":"t01",
+                          |            "Year":2006,
+                          |            "Scene":"sc01",
+                          |            "Flight":60426,
+                          |            "GEO Ver":"ort",
+                          |            "RDN Ver":"c",
+                          |            "Comments":"Alt = 21Kft<br>SOG = 103 kts<br>CLEAR !!",
+                          |            "NASA Log":"6T010",
+                          |            "Rotation":0,
+                          |            "datetime":"2006-04-26T17:52:00Z",
+                          |            "Flight ID":"f060426t01",
+                          |            "Site Name":"Red Mtn Pass 1, CO",
+                          |            "Pixel Size":2.1,
+                          |            "Flight Scene":"f060426t01p00r03_sc01",
+                          |            "Investigator":"Thomas Painter",
+                          |            "Solar Azimuth":139.9,
+                          |            "Number of Lines":6688,
+                          |            "Solar Elevation":60.21,
+                          |            "File Size (Bytes)":7475366912,
+                          |            "Number of Samples":1335,
+                          |            "Max Scene Elevation":4097.59,
+                          |            "Min Scene Elevation":3163.91,
+                          |            "Mean Scene Elevation":3680.71,
+                          |            "Gzip File Size (Bytes)":2673260903
+                          |         }
+                          |      }
+                          |   ]
+                          |}
+                          |""".stripMargin).valueOr(throw _)),
+            StatusCode.Ok,
+            ""
+          )
+        )
+      }
+
+  describe("StacClientSpec") {
+    it("SttpBackendStub collections") {
+      SttpStacClient(backend, uri"http://localhost:9090").collections
+        .map(_.map(_.id) shouldBe "aviris_2006" :: Nil)
+    }
+
+    it("SttpBackendStub items") {
+      SttpStacClient(backend, uri"http://localhost:9090")
+        .items(NonEmptyString.unsafeFrom("aviris_2006"))
+        .map(_.map(_.id) shouldBe "aviris_f060426t01p00r03_sc01" :: Nil)
+    }
+
+    ignore("AsyncHttpClientCatsBackend") {
+
+      val res = AsyncHttpClientCatsBackend[IO]().flatMap { backend =>
+        val client = SttpStacClient(backend, uri"http://localhost:9090")
+        client.collections
+      }
+
+      res map (_ shouldNot be(Empty))
+    }
+
+    ignore("Http4sBackend") {
+      val res = Blocker[IO].flatMap(Http4sBackend.usingDefaultClientBuilder[IO](_)).use { backend =>
+        val client = SttpStacClient(backend, uri"http://localhost:9090")
+        client.collections
+      }
+
+      res map (_ shouldNot be(Empty))
+    }
```

Comment:
I would like to keep it as a short example of how it can be used with different backends.

Reply:
Can you include a comment here with links to relevant docs if that's the case? I don't know anything about what we're supposed to get from different backends and all things equal I have a preference against ignored tests